cli/src/main/resources/application.conf

```diff
@@ -22,6 +22,7 @@ msbConfig {
     # -1 means unlimited
     consumerThreadPoolQueueCapacity = 20
     requeueRejectedMessages = true
+    prefetchCount = 0
```

Comment:
prefetchCount - It's not a key. We don't use AutoAcknowledgement confituration option. Only explicit AmqpAcknowledgementHandler.setAutoAcknowledgement(boolean autoAcknowledgement) method is used for that.